src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/HotReload/ProjectHotReloadSession.cs

```diff
@@ -186,7 +200,7 @@ public async ValueTask StopAsync(CancellationToken cancellationToken)
 
     public ValueTask<bool> SupportsRestartAsync(CancellationToken cancellationToken)
     {
-        return new ValueTask<bool>(_callback.SupportsRestart);
+        return new ValueTask<bool>(_launchProvider is LaunchProfilesDebugLaunchProvider && _launchProfile is not null && _debugLaunchOptions.HasValue);
```

Comment:
Does anything use `IProjectHotReloadSessionCallback.SupportsRestart` anymore? If not, can it be removed?

Or should this logic be moved to the callback?

Reply:
It's not used anymore, I'll remove it